### PR TITLE
Log runtime estimator units

### PR DESCRIPTION
**Pull request by mvpatel2000**
# What does this PR do?

Log runtime estimator units

### CODE REVIEW

File: composer/callbacks/runtime_estimator.py [hunk 1]
```diff
@@ -64,6 +66,7 @@ def __init__(self, skip_batches: int = 1, time_unit: str = 'hours') -> None:
         self.start_dur = None
         self.train_dataloader_len = None
 
+        self.time_unit = time_unit
```
Comment (aspfohl) [inline composer/callbacks/runtime_estimator.py:69]: Recommend this, or updating line 111 to only log when its not none
```suggestion
        self.time_unit = time_unit or 'hours'
```

**Review by aspfohl — APPROVED** [review]
`time_unit` will only be 'seconds', 'minutes', 'hours', or 'days' right? Not any of the values of `TimeUnit` in composer core?

File: composer/callbacks/runtime_estimator.py (hunk 1 above)
Comment (aspfohl) [reply composer/callbacks/runtime_estimator.py:69]: Oh wait I'm dumb, `time_unit: str = 'hours'` should be good enough above 😛

File: composer/callbacks/runtime_estimator.py [hunk 2]
```diff
@@ -48,6 +48,8 @@ class RuntimeEstimator(Callback):
     +===================================+=========================================================+
     | `time/remaining_estimate`         | Estimated time to completion                            |
     +-----------------------------------+---------------------------------------------------------+
+    | `time/remaining_estimate_unit`    | Unit of time specified by user                          |
```
Comment (dakinggg) [inline composer/callbacks/runtime_estimator.py:51]: list the options in the docs here

**Review by dakinggg — COMMENTED** [review]
Unit test and manual test please

**Pull request merged**